src/components/Compare/CompareTableRow.tsx

```diff
@@ -59,17 +71,30 @@ const CompareTableRow = (props: {
     ? location.locationInfo.county.replace('Parish', 'Par.')
     : location.locationInfo.county.replace('County', 'Co.');
 
+  const roundedPopulation = (
+    Math.round(location.locationInfo.population / 100) * 100
+  ).toLocaleString();
```

Comment:
Maybe you can use [`formatEstimate`](https://github.com/covid-projections/covid-projections/blob/develop/src/common/utils/index.ts#L94) which rounds up numbers to a given precision `formatEstimate(population, 2)`.